addOns/network/src/main/java/org/zaproxy/addon/network/ExtensionNetwork.java

```diff
@@ -172,6 +192,13 @@ public class ExtensionNetwork extends ExtensionAdaptor implements CommandLineLis
     private ConnectionOptions connectionOptions;
     private ConnectionOptionsPanel connectionOptionsPanel;
 
+    private ImageIcon rateLimitIcon =
+            new ImageIcon(ExtensionNetwork.class.getResource("resources/ratelimit.png"));
```

Comment:
Will this cause problems in daemon mode? Especially in headless environments.
I know it has in other cases...

Reply:
It will.